src/main/kotlin/domain/Board.kt

```diff
@@ -2,15 +2,57 @@ package domain
 
 class Board private constructor(val rows: List<Row>) {
 
-    val mineCount = rows.flatMap { it.cells }.count { it is Mine }
+    private val allCells: List<Cell> = rows.flatMap { it.cells }
 
-    val cellCount = rows.first().size * rows.size
+    val mineCount: Int
+        get() = allCells.count { it is Mine }
+
+    val cellCount: Int
+        get() = allCells.size
+
+    fun getCell(coordinate: Coordinate): Cell =
+        allCells.find { it.equalsBy(coordinate) } ?: throw IllegalArgumentException("존재하지 않는 cell 입니다")
 
     fun mineCount(cell: Cell): Int =
-        rows.flatMap { it.cells }.count { it is Mine && it.isAdjacentTo(cell) }
+        allCells.count { it.isAdjacentTo(cell) && it is Mine }
 
-    companion object {
+    fun open(cell: Cell): GameStatus {
+        cell.open()
+
+        when (cell) {
+            is Mine -> return GameStatus.LOST
+            is Empty -> getZeroMineCells(cell).forEach { it.openAll() }
+        }
+
+        return if (isClear()) GameStatus.WIN
+        else GameStatus.CONTINUE
```

Comment:
중괄호가 없으니, `else` 부분은 리턴과는 연관이 있다고 바로 파악하기 어려운거같아요!
아래와 같은건 어떨까요?
```suggestion
        return when(isClear()) {
            true -> GameStatus.WIN
            false -> GameStatus.CONTINUE
        }
```